<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>279</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>279</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/images/100.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_page">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>  登录  </string>
    </attribute>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>13252523396</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_login">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>  注册  </string>
    </attribute>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_register">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>输入8-11 位 手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>输入10位以内 密码</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_confirm_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>再次 确认 密码</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_confirm_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tw_page</tabstop>
  <tabstop>le_tel</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>pb_login</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>le_tel_register</tabstop>
  <tabstop>le_password_register</tabstop>
  <tabstop>le_confirm_register</tabstop>
  <tabstop>pb_register</tabstop>
  <tabstop>pb_clear_register</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
